src/test_helpers.rs

```diff
@@ -1,14 +1,5 @@
 use burn::backend::autodiff::Autodiff;
 pub type NdArrayAutodiff = Autodiff<burn::backend::NdArray>;
-use burn::tensor::{Element, Float, TensorData};
-
 pub type Model = crate::model::Model<NdArrayAutodiff>;
-pub type Tensor<const D: usize, K = Float> = burn::tensor::Tensor<NdArrayAutodiff, D, K>;
-
```

Comment:
add `#[track_caller]` here, that's all

ref <https://rustc-dev-guide.rust-lang.org/backend/implicit-caller-location.html>